tests/external-links.js

```diff
@@ -374,22 +371,21 @@ async function updateGithubIssue(urlResults) {
       ``,
       `**Last updated:** ${(new Date()).toISOString()}`,
       ``,
-      `| URL | today | -1d | -2d | -3d | -4d | -5d | -6d |`,
-      `|-----|-------|-----|-----|-----|-----|-----|-----|`,
+      `| URL | today … 6 days ago |`,
+      `|-----|--------------------|`,
       ...Object.entries(linkData).map(([url, statuses]) => {
-        const columns = [
-          url,
-          ...statuses.map(status => {
-            if (!status.failed) {
-              return `:heavy_check_mark:`;
-            }
-
-            const message = status.message.replace(`\n`, ` `).replace(`"`, `&quot;`);
-            return `<a href="${status.jobUrl}" title="${message}">:x:</a>`;
-          })
-        ];
-
-        return `| ${columns.join(` | `)} |`;
+        let str = `| ${url} <td nowrap>`;
+        str += statuses.map(status => {
```

Comment:
Better put the mapped array in a const `statusIcons` and build the string with a single template string directly in the return statement.